lib/rex/socket.rb

```diff
@@ -809,10 +828,64 @@ def type?
 
 protected
 
-  attr_writer :peerhost, :peerport, :localhost, :localport # :nodoc:
+  attr_writer :peerhost, :peerhostname, :peerport, :localhost, :localport # :nodoc:
   attr_writer :context # :nodoc:
   attr_writer :ipv # :nodoc:
 
+  def self.rex_gethostbyname(name, resolver = @@resolver)
```

Comment:
Missing YARD documentation here, which would be quite helpful, particularly for understanding what these parameters types are meant to be.

Reply:
Since been resolved.